apps/openmw/mwphysics/physicssystem.cpp

```diff
@@ -249,7 +249,7 @@ namespace MWPhysics
                 // Check if we actually found a valid spawn point (use an infinitely thin ray this time).
                 // Required for some broken door destinations in Morrowind.esm, where the spawn point
                 // intersects with other geometry if the actor's base is taken into account
-                btVector3 from = toBullet(position);
+                btVector3 from = toBullet(position + offset);
```

Comment:
This kind of change (and the other related ones) make me anxious because we had a lot of problems with various spawn points in Morrowind.esm not working correctly. If you're going to modify the spawn placement then please go through all the related issues we had on the bug tracker (git blame will probably find most of them) and test if they don't reoccur.

Reply:
A quick test did not show anything critical.
Actually, this change fixed placing actors on slopes for me (remember issues in Maar Gan?).
Now we cast ray from actor basement down, so such raytesting sometimes fails to detect ladders and traces actor down to terrain. If the cell is interior, actor can fall infinitely.
This PR uses an actor center as ray origin and detects slopes correctly.
Also this change required for sitting actors - raytest from basement ofter fails to detect furniture:
https://i.imgur.com/6jEvCcG.jpg

Again, my testing results are subjective.